execute/tokendata/http/http.go

```diff
@@ -16,18 +15,13 @@ import (
 
 	"github.com/smartcontractkit/chainlink-common/pkg/logger"
 
+	"github.com/smartcontractkit/chainlink-ccip/execute/tokendata"
+
 	"github.com/smartcontractkit/chainlink-ccip/pkg/logutil"
 	cciptypes "github.com/smartcontractkit/chainlink-ccip/pkg/types/ccipocr3"
 )
 
 const (
-	apiVersion      = "v1"
-	attestationPath = "attestations"
-
-	// defaultCoolDownDurationSec defines the default time to wait after getting rate limited.
```

Comment:
Could we preserve this comment somewhere close to the cooldown values? I think it was very valuable to better understand why this value is needed